doc/distrib/NodeHelpFiles/_review/Dynamo.Nodes.DSFaceSelection.md

```diff
@@ -0,0 +1,8 @@
+## In Depth
+`Select Face` is a user interface node that provides a single selection of a face of a Revit element.
```

Comment:
Like the clarity available in the select edge node description regarding how user can interact with it. So suggest adding that for clarity here.  "Select Face is a user interface node that displays a cursor in Revit that allows you to select a single face"